### PR TITLE
Spring 1.5.x compatibility

### PR DESCRIPTION
**Pull request by lrkwz**
I noticed that you should add a dependency in liquibase-maven-plugin in order to execute when using spring-boot 1.5.x

### CODE REVIEW

**Comment by pavankjadda** [comment]
I will test the changes by EOW and merge them

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# Liquibase database evolution with with Spring Boot 
-Liquibase can not generate ChangeLog from empty database or JPA entities. It should be run against existing database(dev or test) initially before using Liquibase-Hibernate plugin. If you do not have existing database, create new one with 1 table and Liquibase takes it from there. Steps to setup Liquibase from Scratch
+# Liquibase database evolution with with Spring Boot
+Steps to setup Liquibase from Scratch
```
Comment (pavankjadda) [inline README.md]: The following is needed 

```
Liquibase can not generate ChangeLog from empty database or JPA entities. It should be run against existing database(dev or test) initially before using Liquibase-Hibernate plugin. If you do not have existing database, create new one with 1 table and Liquibase takes it from there. Steps to setup Liquibase from Scratch
```

**Comment by lrkwz** [comment]
I merged your master and developement branches into the fork

**Comment by lrkwz** [comment]
Sorry I didn't want to close the merge request

File: README.md (hunk 1 above)
Comment (lrkwz) [reply README.md]: I merged your sources; both branches should auto merge with no problems

**Pull request merged**